process/interface.go

```diff
@@ -71,11 +68,11 @@ type DBMiniblocksHandler interface {
 	SerializeBulkMiniBlocks(bulkMbs []*data.Miniblock, mbsInDB map[string]bool) *bytes.Buffer
 }
 
-// DBGeneralInfoHandler defines the actions that a general info handler should do
-type DBGeneralInfoHandler interface {
-	PrepareGeneralInfo(tpsBenchmark statistics.TPSBenchmark) (*data.TPS, []*data.TPS)
+// DBStatisticsHandler defines the actions that a general info handler should do
```

Comment:
... that a **database statistics** handler should do

Reply:
done.